src/components/style/interactive/dialog.css

```diff
@@ -0,0 +1,68 @@
+.dialog-enter {
+  opacity: 0;
+}
+
+.dialog-enter-active {
+  opacity: 1;
+  transition: opacity 200ms;
+}
+
+.dialog-exit {
+  opacity: 1;
+}
+
+.dialog-exit-active {
+  opacity: 0;
+  transition: opacity 200ms;
+}
+
+.dialog-overlay {
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  position: fixed;
+  /* leaving the zero addition here for now */
+  z-index: calc(var(--zindex-modal) + 0);
```

Comment:
we have a `--zindex-shim` for this - do you think that could work well enough?

Reply:
I think so. I'll adjust.